features/lwipstack/mbed_lib.json

```diff
@@ -128,6 +128,10 @@
         "ppp-thread-stacksize": {
             "help": "Thread stack size for PPP (obsolete: use netsocket/ppp configuration instead)",
             "value": 768
+        },
+        "rawipsocket-enabled": {
```

Comment:
Other option names would suggest definitely a `-` before `socket`, and maybe between `raw` and `ip`. Could also be `raw-socket-enabled`. Not sure `ip` is needed.